### DISCUSSION TITLE
Toast Click Action + Layout

### DISCUSSION THREAD

**Discussion by bindalf** [CommunityToolkit/Maui#2154] [General]
Hello, how can you set the action if user clicks on a Toast. I only tested on Windows but when i click on the Toast it opens another instance of my application. 
and Is the the layout adjustable?
I have seen Toasts of other application where  a picture is displayed and also a custom action is set e.g. open a file explorer and so on.

**Comment by ghost** [comment]
@bindalf according to the [docs](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/alerts/toast?tabs=windows%2Candroid), you can not. Maybe if you would create a small popup and add a picture and a custom action e.g. open a file explorer, this could work. [Community toolkit Popup is really nice](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/popup).
